libbeat/processors/decode_xml/docs/decode_xml.asciidoc

```diff
@@ -113,3 +114,78 @@ when a specified field does not exist. Defaults to `false`.
 Defaults to `false`.
 
 See <<conditions>> for a list of supported conditions.
+
+
+==== Schemas
+
+When a schema is defined, the specific decoder will parse the configured field.
+The ouput of the parsing will be specific to that schema.
+
+===== Wineventlog
+
+The `wineventlog` schema decodes Windows Events. The resulting fields under the specified target field
```

Comment:
It might just be me that likes this, but I really like to have these paragraph lines wrapped at 80 when dealing with asciidoc.